cms/models_test.py

```diff
@@ -270,6 +275,19 @@ def test_program_page_faculty_subpage():
     _assert_faculty_members(program_page)
 
 
+def test_external_program_page_faculty_subpage():

```

Comment:
Am I missing something, what is the relation of external ProgramPages with faculty subpages ?

Reply:
Just like the above comment, `Faculty Page` page is basically a subpage of the Product Page itself which serves as the parent for all our courseware pages `Courses, External Courses, Programs`(Tests for these pages are added already), hence `Faculty` subpage is also supported in External Program page since it also serves as a subpage to the product page, This test just adds the compatibility check for External Programs